Dockerfile

```diff
@@ -31,9 +31,9 @@ RUN set -x \
   && rm -rf /var/lib/apt/lists/* \
   # Post-install java certificates
   /var/lib/dpkg/info/ca-certificates-java.postinst configure \
-  # Install npm dependencies
+  # Install npm dependencies (System Tests!)
   && cd /dcos-ui \
-  && npm install -g junit-merge@1.3.0 git://github.com/johntron/http-server.git#proxy-secure-flag \
+  && npm install -g git://github.com/johntron/http-server.git#proxy-secure-flag \
```

Comment:
Do we still need this if it's in the package.json?

Reply:
Yeah, system tests rely on it :(